### PR TITLE
stream-processing-client.adoc: wrong description of TestSources.itemStream

### PR DESCRIPTION
**Pull request by Petkomat**
In `TestSources.itemStream(n)`, `n` is the number of items per second and not (as stated previously), the number of seconds per item.

### CODE REVIEW

**Review by Serdaro — APPROVED** [review]

**Comment by Serdaro** [comment]
Thanks a lot for the fix @Petkomat , you can ignore the link checker warnings and merge the PR.
The fix will also be applied to all 5.x documentation series.

**Comment by Petkomat** [comment]
@Serdaro I cannot merge the pull request (The message above says: Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.)

**Pull request merged**
